permit.md

```diff
@@ -52,7 +56,7 @@ select * from vps_vessel where (date_issued<>date_canceled or date_canceled is n
 
 
 # Related Tables
-+ BUS_OWN contains ownership data that is often linked to these PERMIT data. We'll put them in a separate section.
```

Comment:
Lets leave this in here (for now), but say something like

 + BUS_OWN contains ownership data that is often linked to these PERMIT data. However, the data is stale. It has not been updated since 2017, so, don't use it. (April 14, 2022).

Reply:
https://github.com/NEFSC/READ-SSB-Lee-metadata/blob/master/Client.md
The above is consistent for me, feel free to add your comment above.